src/time_bucket.c

```diff
@@ -402,6 +437,25 @@ ts_date_bucket(PG_FUNCTION_ARGS)
 	}
 }
 
+TS_FUNCTION_INFO_V1(ts_date_offset_bucket);
+
+TSDLLEXPORT Datum
+ts_date_offset_bucket(PG_FUNCTION_ARGS)
+{
+	Datum period = PG_GETARG_DATUM(0);
+	Datum date = PG_GETARG_DATUM(1);
+	/* Apply offset. */
```

Comment:
Nit: The other functions are formatted in a slightly different way (e.g., you have an empty line before `/* Apply offset. */` and no empty line before `/* Remove offset. */`). Could you use consistent formatting? Then it is easier to spot the differences between the functions.

Reply:
Changed